packages/core/stories/styles/index.scss

```diff
@@ -39,6 +39,9 @@
 @use '@massds/mayflower-assets/scss/02-molecules/footer-links';
 @use '@massds/mayflower-assets/scss/02-molecules/social-links';
 
+@use '@massds/mayflower-assets/css/layout.css';
+@use '~@massds/mayflower-assets/scss/01-atoms/place-holder';
```

Comment:
Why does this line need the `~` but the others do not?

Reply:
oh it doesn't need `~` because we have included node_modules path already